DESCRIPTION

```diff
@@ -19,8 +19,8 @@ Authors@R: c(
     person("Dony", "Unardi", , "unardid@gene.com", role = "rev"),
     person("F. Hoffmann-La Roche AG", role = c("cph", "fnd"))
   )
-Description: Utilities for data transformations compatible with 'teal'
-    applications.
+Description: Utilities to provide a unified data model for `teal` applications
```

Comment:
👍

Reply:
This is not that different from what we had, I phrased similarly to `tidyverse` descriptions, but I'm thinking it doesn't address part of the comment

> Please add more details about the package functionality and \***implemented methods**\* in your
Description text. (emphasis mine)

Reply:
Small tweak to mention the `teal_data` class in the spirit of `tibble`

> Description: Provides a 'tbl_df' class (the 'tibble') with stricter checking and better formatting than the traditional data frame.

```diff
-Utilities to provide a unified data model for teal applications focusing on reproducibility and relational data.
+Provides a `teal_data` class as a unified data model for `teal` applications focusing on reproducibility and relational data.
```

Reply:
unified data _container_?

Reply:
Good & accurate suggestion :smile: 

I wrote _data model_ as it supports a relational model with `join_keys` (and it's a nice buzzword). I'm not opposed to change it to container though